dlib/test/rand.cpp

```diff
@@ -413,7 +413,7 @@ namespace
         print_spinner();
         dlib::rand rnd(0);
         
-        const size_t N = 1024*1024*4;
+        const size_t N = 1024*1024*16;
```

Comment:
Does this need to be increased?

Reply:
I thought i reverted that back to what it was before. To answer your question, if it passes the test then no. Unfortunately, measuring kurtosis and skewness requires a lot of samples. Maybe there could be other, equally meaningful yet faster unit tests for rand. But that should be dealt with in another PR maybe.

Reply:
Don't forget to revert this.